Benp/add changelog

benpatterson opened the pull request.

benpatterson commented:
@martenson @bramwelt this is in response to issues you logged; if you have time for a review, I'd appreciate your feedback. I did some lightweight grok'ing of the commit history to construct the info after 1.30.2

martenson commented on CHANGELOG.md:
```diff
@@ -0,0 +1,231 @@
+#### -> 1.32.1
+* Allow for manual managing of webhooks
+* Various fixes, like:
+** Avoid overwriting test results when there's a race condition
```
this and few others below below have extra asterisks that render incorrectly

benpatterson replied:
done

martenson commented on CHANGELOG.md:
```diff
@@ -0,0 +1,231 @@
+* Don't check for comments when a new PR is opened.
+
+#### -> 1.32.3
+* Set the paramters again.
```
s/paramters/parameters

benpatterson replied:
done

benpatterson commented:
Thanks @martenson I addressed your comments and will go ahead and merge now.

We can go through a second round of updates if there's still confusion, but this gets us a relatively up-to-date changelog and that's cool.

martenson commented on CHANGELOG.md:
```diff
@@ -0,0 +1,214 @@
+* Don't check for comments when a new PR is opened.
+
+#### -> 1.32.3
+* Set the paramaters again.
```
This fix did not work out. 😆

benpatterson replied:
haha. oh noes you're right!

I'll put in a quick PR to fix it (again!)
